src/test/java/AccountTest.java

```diff
@@ -25,6 +25,7 @@
 
     @Test
     public void get_all_account_successful_200() {
+        apiRequest.clearPathParams();
```

Comment:
The method clearPathParams doesnt exist.

Reply:
Ok Gus, I ready fired